pkg/apis/hco/v1alpha1/hyperconverged_types.go

```diff
@@ -69,3 +73,43 @@ type HyperConvergedList struct {
 func init() {
 	SchemeBuilder.Register(&HyperConverged{}, &HyperConvergedList{})
 }
+
+type KubevirtConfigurations struct {
+	ResourceVersion                   string            `json:"ResourceVersion,omitempty"`
+	UseEmulation                      bool              `json:"UseEmulation,omitempty"`
```

Comment:
The json names should all start lower-case. `useEmulation` instead of `UseEmulation` for instance.